src/__tests__/pwnedPassword.test.ts

```diff
@@ -19,7 +19,7 @@ describe('pwnedPassword', () => {
     it('resolves to 0', () => {
       server.use(
         rest.get('*', (_, res, ctx) => {
-          return res(ctx.json(null));
+          return res(ctx.text(EXAMPLE_PASSWORD_HASHES));
```

Comment:
Was this change made because there should always be matching suffixes returned? Just curious.

Reply:
When I first read it I was confused by `ctx.json(null)`, so I switched to `ctx.text('')`. Having done that, I thought it would be a more thorough test if there _were_ suffixes in the response, but none matched the test case.

Reply:
Gotcha. I originally had that because (I think?) I assumed the remote API responded that way when there were no suffixes, but upon re-reading their docs, it sounds like there will always be data. Good change here, regardless. 👍